esp/src/eclwatch/nls/es/hpcc.js

```diff
@@ -260,6 +264,7 @@ define(
         Fetched: "Obtenido",
         FetchingData: "Obteniendo Data...",
         fetchingresults: "obteniendo resultados",
+        FieldNames: "Nombre de campos",
```

Comment:
It feels like this should be "Nombres de campos" hard to tell w/out the English text

Reply:
This is a tough one. I have seen for "input fields": campos de entrada

Reply:
100% agreed: "Campos de entrada" is correct for input fields, I'm not sure that's what the variable "FieldNames" is trying to convey. Can we get the original English phrase for context?

Reply:
English is just "Field Names" just to define the title bar for a field of bloom names